### PR TITLE
Cloudfront

### PR DESCRIPTION
**Pull request by madelynnblue**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,30 +1,13 @@
...
+WORKDIR /srv/jekyll
 
-# Set runtime options
-EXPOSE 4000
-CMD ["jekyll", "serve", "--host", "0.0.0.0"]
+COPY run.sh /
+
+ADD . /srv/jekyll
+
+CMD ["sh", "/run.sh"]
```
Comment (jakedt) [inline Dockerfile]: The default should let you run this in local mode, not deploy. Jenkins should fill in the deploy commands directly.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+set -e
+set -x
+
+jekyll build
+aws s3 sync ./_site s3://quay-docs --delete --acl public-read
+aws configure set preview.cloudfront true
+aws cloudfront create-invalidation --distribution-id E26ZSD57Z81FLL --invalidation-batch "{\"Paths\":{\"Quantity\":1,\"Items\":[\"/*\"]},\"CallerReference\":\"jenkins-$(date)\"}"
```
Comment (jakedt) [inline run.sh]: Can we find definitively if the distribution id is a secret?

Comment (madelynnblue) [inline run.sh]: I've changed it to a var, but it's still out here in the outdated PR change. I can't find documentation on whether or not it's a secret.

Comment (josephschorr) [inline run.sh]: We should rotate it anyway if we can (and this discussion is, of course, public)

Comment (madelynnblue) [inline run.sh]: Rotating it means taking down the entire cloudfront distribution and recreating it, bringing down the docs site.

File: Dockerfile (hunk 1 above)
Comment (jakedt) [inline Dockerfile]: Ping? Are there larger changes for this PR related to how difficult CF distributions are to work with this way?

**Comment by madelynnblue** [comment]
Changed so that the default is to serve, and jenkins can deploy however it needs.

**Comment by jakedt** [comment]
LGTM

**Pull request merged**